rules/permissions.py

```diff
@@ -17,6 +17,7 @@ def perm_exists(name):
 
 
 def has_perm(name, *args, **kwargs):
+    kwargs['perm'] = name
```

Comment:
`rules` is more than Django permissions, so `kwargs` should be augmented [here](https://github.com/dfunckt/django-rules/blob/master/rules/rulesets.py#L6), and the attribute should be called `rule`, instead of `perm`.